haskell/private/actions/compile.bzl

```diff
@@ -258,7 +272,7 @@ def _compilation_defaults(hs, cc, java, posix, dep_info, plugin_dep_info, srcs,
         # to debug issues in non-sandboxed builds.
         "-Wmissing-home-modules",
     ])
-    if hs.toolchain.is_static and not hs.toolchain.is_windows:
+    if hs.toolchain.static_runtime and not hs.toolchain.is_windows:
         # A static GHC RTS requires -fPIC. However, on Unix we also require
         # -fexternal-dynamic-refs, otherwise GHC still generates R_X86_64_PC32
         # relocations which prevents loading these static libraries as PIC.
```

Comment:
Looking at this I think I should check for `is_windows` in the Cabal wrapper too, correct? I don't do this at present so will revisit.

Reply:
Yes, IIRC `-fexternal-dynamic-refs` doesn't work on Windows, or at least is not requried. Preferably, that would go into `haskell/cabal.bzl`.